### DISCUSSION TITLE
Can Magpie be installed in OpenHPC with Lustre

### DISCUSSION THREAD

**Discussion by lingforrest** [llnl/magpie#333] [General]
Is it possible that Magpie can be installed on OpenHPC environment with Lustre FS ? OpenHPC website: http://openhpc.community/

**Comment by chu11** [comment]
The HPC software stack isn't as important, it's more the software that is on the HPC stack.  You'll need one of the schedulers / resource managers supported by Magpie and a parallel file system (which Lustre should be fine).  If you take a look at `doc/README` you'll see a list of requirements.

**Comment by rpomaris** [comment]
@lingforrest I can help with this - Intel has a full Install Guide to help stand up Magpie on an openHPC system, and we have used it with Lustre internally.

**Reply by pripiat1986** [reply]
Hello mepowers,  Could you provide me the link of the Intel guide for Magpie in OpenHPC ? Thanks.

**Comment by jcsiadal** [comment]
Thanks, @mepowers. Magpie has been officially added to OpenHPC and I just started working on the OpenHPC 2.0 package for it. I'll be pestering your team for help shortly.
